File: phone-append-2-nodejs-examples/business_phone_append_soap_example.js
import { BusinessPhoneAppendSoap } from '../phone-append-2-nodejs/SOAP/business_phone_append_soap.js';

async function businessPhoneAppendSoapExample(isLive, licenseKey) {
    console.log("\n-------------------------------------------------");
    console.log("PhoneAppend2 - BusinessPhoneAppend - SOAP Example");
    console.log("-------------------------------------------------");

    const businessName = "Brooks Institute";
    const address = "27 E Cota ST";
    const city = "Santa Barbara";
    const state = "CA";
    const postalCode = "93101";
    const timeoutSeconds = 15;

    console.log("\n* Input *\n");
    console.log(`Business Name  : ${businessName}`);
    console.log(`Address        : ${address}`);
    console.log(`City           : ${city}`);
    console.log(`State          : ${state}`);
    console.log(`Postal Code    : ${postalCode}`);
    console.log(`License Key    : ${licenseKey}`);
    console.log(`Is Live        : ${isLive}`);
    console.log(`Timeout Seconds: ${timeoutSeconds}`);

    try {
        const pa2 = new BusinessPhoneAppendSoap(
            businessName,
            address,
            city,
            state,
            postalCode,
            licenseKey,
            isLive,
            timeoutSeconds
        );
        const response = await pa2.invokeAsync();

        console.log("\n* Phone Info *\n");
        if (response) {
            if (response.PhoneInfo) {
                console.log(`Phone         : ${response.PhoneInfo.Phone}`);
                console.log(`Name          : ${response.PhoneInfo.Name}`);
                console.log(`Address       : ${response.PhoneInfo.Address}`);
                console.log(`City          : ${response.PhoneInfo.City}`);
                console.log(`State         : ${response.PhoneInfo.State}`);
                console.log(`Postal Code   : ${response.PhoneInfo.PostalCode}`);
                console.log(`Is Residential: ${response.PhoneInfo.IsResidential}`);
                console.log(`Certainty     : ${response.PhoneInfo.Certainty}`);
                console.log(`Line Type     : ${response.PhoneInfo.LineType}`);
            } else {
                console.log("No phone info found.");
            }
        } else {
            console.log("No phone info found.");
        }

        if (response.Error) {
            console.log("\n* Error *\n");
            console.log(`Error Type     : ${response.Error.Type}`);
            console.log(`Error Type Code: ${response.Error.TypeCode}`);
            console.log(`Error Desc     : ${response.Error.Desc}`);
            console.log(`Error Desc Code: ${response.Error.DescCode}`);
        }
    } catch (e) {
        console.log("\n* Error *\n");
        console.log(`Error Message: ${e.message}`);
    }
}

export { businessPhoneAppendSoapExample };

File: phone-append-2-python-examples/business_phone_append_soap_example.py
import sys
import os

sys.path.insert(0, os.path.abspath("../phone-append-2-python/SOAP"))

from business_phone_append_soap import BusinessPhoneAppendSoap

def business_phone_append_soap_sdk_go(is_live: bool, license_key: str) -> None:
    print("\r\n---------------------------------------------")
    print("PhoneAppend2 - BusinessPhoneAppend - SOAP SDK")
    print("---------------------------------------------")

    business_name = "Brooks Institute"
    address = "27 E Cota ST"
    city = "Santa Barbara"
    state = "CA"
    postal_code = "93101"
    timeout_seconds = 15

    print("\r\n* Input *\r\n")
    print(f"Business Name  : {business_name}")
    print(f"Address        : {address}")
    print(f"City           : {city}")
    print(f"State          : {state}")
    print(f"Postal Code    : {postal_code}")
    print(f"License Key    : {license_key}")
    print(f"Is Live        : {is_live}")
    print(f"Timeout Seconds: {timeout_seconds}")

    try:
        service = BusinessPhoneAppendSoap(license_key, is_live, timeout_seconds * 1000)
        response = service.get_business_phone_append(business_name, address, city, state, postal_code)

        if not hasattr(response, 'Error') or not response.Error:
            print("\r\n* Phone Info *\r\n")
            if hasattr(response, 'PhoneInfo') and response.PhoneInfo:
                print(f"Phone           : {getattr(response.PhoneInfo, 'Phone', None)}")
                print(f"Name            : {getattr(response.PhoneInfo, 'Name', None)}")
                print(f"Address         : {getattr(response.PhoneInfo, 'Address', None)}")
                print(f"City            : {getattr(response.PhoneInfo, 'City', None)}")
                print(f"State           : {getattr(response.PhoneInfo, 'State', None)}")
                print(f"Postal Code     : {getattr(response.PhoneInfo, 'PostalCode', None)}")
                print(f"Is Residential  : {getattr(response.PhoneInfo, 'IsResidential', None)}")
                print(f"Certainty       : {getattr(response.PhoneInfo, 'Certainty', None)}")
                print(f"Line Type       : {getattr(response.PhoneInfo, 'LineType', None)}")
            else:
                print("No phone info found.")
        else:
            print("No phone info found.")

        if hasattr(response, 'Error') and response.Error:
            print("\r\n* Error *\r\n")
            print(f"Error Type      : {getattr(response.Error, 'Type', None)}")
            print(f"Error Type Code : {getattr(response.Error, 'TypeCode', None)}")
            print(f"Error Desc      : {getattr(response.Error, 'Desc', None)}")
            print(f"Error Desc Code : {getattr(response.Error, 'DescCode', None)}")

    except Exception as e:
        print("\r\n* Error *\r\n")
        print(f"Exception occurred: {str(e)}")